packages/manager/modules/pci/src/projects/project/instances/instances.routing.js

```diff
@@ -80,83 +80,165 @@ export default /* @ngInject */ ($stateProvider) => {
         $state.go('pci.projects.project.instances.add', {
           projectId,
         }),
-      viewInstance: /* @ngInject */ ($state, projectId) => (instance) =>
-        $state.go('pci.projects.project.instances.instance', {
-          projectId,
-          instanceId: instance.id,
+      trackingPrefix: /* @ngInject */ () =>
+        `PublicCloud::pci::projects::project::instances`,
```

Comment:
Can be move in a constant file ?

Reply:
It is used only in one place. So I decided to place it here.